### PR TITLE
maintainers details

### PR DESCRIPTION
**Pull request by gianarb**
Reference to https://github.com/zendframework/zf2/issues/7607

### CODE REVIEW

File: TEAMS.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+[EventManager](https://github.com/zendframework/zend-eventmanager) | | |
+[Feed](https://github.com/zendframework/zend-feed) | | |
+[File](https://github.com/zendframework/zend-file) | | |
+[Filter](https://github.com/zendframework/zend-filter) | |
+[Form](https://github.com/zendframework/zend-form) | | |
+[Http](https://github.com/zendframework/zend-http) | | |
+[I18n](https://github.com/zendframework/zend-I18n) | | |
+[InputFilter](https://github.com/zendframework/zend-inputfilter) | | |
+[Json](https://github.com/zendframework/zend-json) | | |
+[Ldap](https://github.com/zendframework/zend-ldap) | | |
```
Comment (weierophinney) [inline TEAMS.md]: For maintainers here, you can use @heiglandreas and @Maks3w

File: TEAMS.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Projects details
+
+| *Project*       | *Maintainers*  | *Teams* |
+| --------------- | -------------- | ------- |
+[ZendSkeletonApplication](https://github.com/zendframework/ZendSkeletonApplication) | | |
+[Authentication](https://github.com/zendframework/zend-authentication) | | |
+[Barcode](https://github.com/zendframework/zend-barcode) | | |
+[Cache](https://github.com/zendframework/zend-cache) | | |
```
Comment (weierophinney) [inline TEAMS.md]: Add @marc-mabe here.

File: TEAMS.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Projects details
+
+| *Project*       | *Maintainers*  | *Teams* |
+| --------------- | -------------- | ------- |
+[ZendSkeletonApplication](https://github.com/zendframework/ZendSkeletonApplication) | | |
```
Comment (weierophinney) [inline TEAMS.md]: Add @Akrabat and @EvanDotPro here.

File: TEAMS.md (hunk 1 above)
Comment (weierophinney) [inline TEAMS.md]: And for team, @zendframework/zend-ldap

File: TEAMS.md (hunk 2 above)
Comment (weierophinney) [inline TEAMS.md]: with team @zendframework/zend-cache

**Comment by weierophinney** [comment]
For all components, you can add the team @zendframework/community-review-team as a team.

**Comment by gianarb** [comment]
for the team I don't use link because return 404.. :) I use only name "community-review-team", "team-ldab"...

**Comment by Maks3w** [comment]
We should put a big note about security issues on top of the page

**Comment by gianarb** [comment]
Security issues about? Sorry I don't understand

**Comment by Maks3w** [comment]
If someone look this table for to report a security issue then he/she should contact with zf-security@zend.com

**Comment by weierophinney** [comment]
@gianarb Put the note about "reporting security issues" found in any `CONTRIBUTING.md` of any component at the top of the file.

File: TEAMS.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Projects details
+
+| *Project*       | *Maintainers*  | *Teams* |
+| --------------- | -------------- | ------- |
+[ZendSkeletonApplication](https://github.com/zendframework/ZendSkeletonApplication) | [Akrabat](https://github.com/akrabat), [EvantDotPro](https://github.com/EvanDotPro) | community-review-team |
```
Comment (weierophinney) [inline TEAMS.md]: Use the notation `@zendframework/{team name}` throughout -- that way they'll be linked.

Comment (weierophinney) [inline TEAMS.md]: Same for contributors - use `@username`; it's shorter and creates the links for you.

Comment (gianarb) [inline TEAMS.md]: @zendframwork/{team name} creates a broken link :(
If I use @username works in github but if you use it outside this site doesn't work..

**Comment by gianarb** [comment]
news? :)

**Comment by marc-mabe** [comment]
@gianarb @weierophinney I thing I'm be missing as maintainer for the `zend-serializer` component. It's a very simple and stable component but initially wrote it and I'm still listed as maintainer in the zf2 wiki.

@weierophinney Where I have write permissions for `zend-cache` I don't have it for `zend-serializer` :(

**Comment by weierophinney** [comment]
@marc-mabe You should be all set now.

@gianarb Merged!

**Comment by gianarb** [comment]
Thanks!

**Comment by marc-mabe** [comment]
@weierophinney thanks !

**Pull request merged**